Unexpected success in e.g. TestExportOpInfoCPU.test_fake_export___getitem___cpu_float32

Same with PYPI package and reported in https://github.com/pytorch/pytorch/pull/164166

Skip all instead of XFailing

Author: Alexander Grund (TU Dresden)

diff --git a/test/export/test_export_opinfo.py b/test/export/test_export_opinfo.py
index 35d8b2895bd..409a305a3aa 100644
--- a/test/export/test_export_opinfo.py
+++ b/test/export/test_export_opinfo.py
@@ -22,54 +22,54 @@ from torch.utils import _pytree as pytree
 
 # following are failing with regular torch.export.export
 export_failures = {
-    xfail("allclose"),
-    xfail("combinations"),
-    xfail("corrcoef"),
-    xfail("cov"),
-    xfail("equal"),
-    xfail("linalg.lstsq"),
-    xfail("linalg.lstsq", "grad_oriented"),
-    xfail("nn.functional.ctc_loss"),
-    xfail("nn.functional.gaussian_nll_loss"),
-    xfail("sparse.sampled_addmm"),
-    xfail("tensor_split"),
+    skip("allclose"),
+    skip("combinations"),
+    skip("corrcoef"),
+    skip("cov"),
+    skip("equal"),
+    skip("linalg.lstsq"),
+    skip("linalg.lstsq", "grad_oriented"),
+    skip("nn.functional.ctc_loss"),
+    skip("nn.functional.gaussian_nll_loss"),
+    skip("sparse.sampled_addmm"),
+    skip("tensor_split"),
 }
 
 # following are failing fake export on cuda device
 fake_export_failures = {
-    xfail("geqrf"),
-    xfail("histogram"),
-    xfail("masked.amax"),
-    xfail("masked.amin"),
-    xfail("masked.argmax"),
-    xfail("masked.argmin"),
-    xfail("masked.logaddexp"),
-    xfail("masked.logsumexp"),
-    xfail("masked.mean"),
-    xfail("masked.prod"),
-    xfail("masked.std"),
-    xfail("masked.sum"),
-    xfail("masked.var"),
-    xfail("nn.functional.grid_sample"),
-    xfail("to_sparse"),
+    skip("geqrf"),
+    skip("histogram"),
+    skip("masked.amax"),
+    skip("masked.amin"),
+    skip("masked.argmax"),
+    skip("masked.argmin"),
+    skip("masked.logaddexp"),
+    skip("masked.logsumexp"),
+    skip("masked.mean"),
+    skip("masked.prod"),
+    skip("masked.std"),
+    skip("masked.sum"),
+    skip("masked.var"),
+    skip("nn.functional.grid_sample"),
+    skip("to_sparse"),
     # cannot xfail as it is passing for cpu-only build
     skip("nn.functional.conv2d"),
     skip("nn.functional.scaled_dot_product_attention"),
     # following are failing due to OptionalDeviceGuard
-    xfail("__getitem__"),
-    xfail("nn.functional.batch_norm"),
-    xfail("nn.functional.instance_norm"),
-    xfail("nn.functional.multi_margin_loss"),
-    xfail("nonzero"),
+    skip("__getitem__"),
+    skip("nn.functional.batch_norm"),
+    skip("nn.functional.instance_norm"),
+    skip("nn.functional.multi_margin_loss"),
+    skip("nonzero"),
 }
 
 fake_decomposition_failures = {
-    xfail("linalg.matrix_rank"),
-    xfail("nn.functional.binary_cross_entropy_with_logits"),
-    xfail("nn.functional.instance_norm"),
-    xfail("nn.functional.multi_margin_loss"),
-    xfail("repeat_interleave"),
-    xfail("take"),
+    skip("linalg.matrix_rank"),
+    skip("nn.functional.binary_cross_entropy_with_logits"),
+    skip("nn.functional.instance_norm"),
+    skip("nn.functional.multi_margin_loss"),
+    skip("repeat_interleave"),
+    skip("take"),
 }
 
 
